freshmaker/errata.py

```diff
@@ -116,14 +118,22 @@ def from_advisory_id(cls, errata, errata_id):
         security_impact = erratum_data["security_impact"].lower()
 
         has_hightouch_bug = False
+        has_compliance_priority_bug = False
         bugs = errata._get_bugs(erratum_data["id"]) or []
         for bug in bugs:
             if "flags" in bug and "hightouch+" in bug["flags"]:
                 has_hightouch_bug = True
+            if "flags" in bug and "compliance_priority+" in bug["flag"]:
+                has_compliance_priority_bug = True
+            if has_hightouch_bug and has_compliance_priority_bug:
                 break
```

Comment:
How about approaching it in this manner:
```
has_hightouch_bug = any("hightouch+" in bug.get("flags", "") for bug in bugs)
has_compliance_priority_bug = any("compliance_priority+" in bug.get("flags", "") for bug in bugs)
```
This makes the code appear more straightforward and easier to comprehend.

Reply:
That's much better!